.code-samples.meilisearch.yaml

```diff
@@ -343,17 +343,16 @@ primary_field_guide_add_document_primary_key: |-
     }
   ], primaryKey: 'reference_number');
 getting_started_add_documents_md: |-
-  ```yaml
-  dependencies:
-    meilisearch: ^0.3.1
+  ```bash
+  dart pub add meilisearch
```

Comment:
we rely on the dart cli to get the latest version instead of maintaining it manually